CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+* Enhancement: Test Native Frames (#793)
```

Comment:
```suggestion
```

We only need user-facing changes here.
You can add `_#skip-changelog_` to the PR description.